STRATEGIC-INITIATIVES.md

```diff
@@ -41,9 +38,11 @@ and have the support needed.
 | This week in Core        | ?                               | https://github.com/nodejs/community-committee/issues/148   |
 
 
-# Completed / Retired
+# Completed / Retired / Inactive
 
 | Initiative        | Champion                        | Links                                                            |
```

Comment:
Would it be pertinent to add a column to indicate the status of the initiative?
While Inactive/Retired initiatives will probably not be reactivated, I think that indicating which ones are "simply" inactive may help drive people toward them to help reactivate.
Or is it just me?

Reply:
I think it would be helpful, especially if they're inactive for want of contributors. I said they're hibernated. 🐻

Made a PR with definitions. And added a column. Feel free to revert.